package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "fightpandemics",
```

Comment:
not sure why you added package.json in root? this was probably moved, did you start from an old outdated branch and tried to put it back  on master?

Reply:
No, not at all, before I start off on any of the tasks, I make it a point of duty to update my local branch with the remote master branch.